Scripts/Mend SCA/README.md

```diff
@@ -323,4 +324,30 @@ curl -o ./whitesource/duediligencereport.xlsx -X POST "${WS_URL}/api/v1.4" -H "C
 ```
 
 <br>
-<hr>
\ No newline at end of file
+<hr>
+
+## Feature Branch Scan
+
+The Unified Agent always uploads a project/scan to the user interface unlike the Mend CLI which has the ability to scan and provide rich output without creating a new project.  To replicate this feature the following must be performed with the UA.  This most commonly used when scanning feature branches or pull requests as these scans should not be retained in the user interface for long periods of time.
+
+**Prerequisites:**  
+* Check that the project state is [finished](check-project-state.sh)
+* `jq` and `awk` must be installed
+* ENV variables must be set
+  * WS_GENERATEPROJECTDETAILSJSON: true
+  * WS_USERKEY
+  * WS_WSS_URL
+
```

Comment:
Add a requirement for:
- WS_APIKEY
- WS_PROJECTNAME
(or something to that effect, these are bare minimum required for Unified Agent).